### PR TITLE
Add link to logo

### PR DESCRIPTION
**Pull request by khd33j**
- [x] This PR adds a link to the logo

    - [x] Set an onClick() listener to the icon (BU) and 'BlueWave Uptime' 
    - [x] The [link](https://github.com/bluewave-labs/bluewave-uptime) opens up in a new tab as specified 

Let me know if I need to make further changes.

### CODE REVIEW

**Comment by gorkem-bwl** [comment]
Can you make a video showing how it behaves when the sidebar is open and closed?

**Comment by khd33j** [comment]
I don't really know how to run the app locally.

**Comment by gorkem-bwl** [comment]
Make sure you test the code locally before sending a PR. 

https://bluewavelabs.gitbook.io/uptime-manager - here you can find the installation documentation.

**Comment by khd33j** [comment]
Noted. I'll remember that for next time. 

I'm gonna reply with a video as soon as I install the app. 
Thanks for the feedback!

File: Client/src/Components/Sidebar/index.jsx [hunk 1]
```diff
@@ -173,7 +173,7 @@ function Sidebar() {
       }}
     >
       <Stack pt={theme.spacing(6)} pb={theme.spacing(12)} pl={theme.spacing(8)}>
-        <Stack direction="row" alignItems="center" gap={theme.spacing(4)}>
+        <Stack direction="row" alignItems="center" gap={theme.spacing(4)} onClick={() => window.open("https://github.com/bluewave-labs/bluewave-uptime", "_blank", "noreferrer")}>
```
Comment (ajhollid) [inline Client/src/Components/Sidebar/index.jsx:176]: Can you add `sx={{ cursor: "pointer" }}` to this stack please?  That's the MUI way of declaring an inline style.

Thanks!

**Review by ajhollid — CHANGES_REQUESTED** [review]
Looks great!  We just need to add `cursor:pointer` to  the `Stack` so that the cusor behaves as if it were a link.

No need to close your PRs and make new ones when changes are requested, you can simply make the changes and push to your branch and your PR will be updated :smile: 

Thanks for your hard work!

**Comment by khd33j** [comment]
@ajhollid Okay, I'll add it in.

I didn't make the changes on a new branch. I used the develop branch itself. 
Figured it wouldn't matter since it is a small change. 

Can I still push it here or should I close the PR and push it in a new one?

**Comment by ajhollid** [comment]
> @ajhollid Okay, I'll add it in.
> 
> I didn't make the changes on a new branch. I used the develop branch itself. 
> Figured it wouldn't matter since it is a small change. 
> 
> Can I still push it here or should I close the PR and push it in a new one?

Ah sorry I didn't notice that.

Yes please always create a new branch when implementing features, even if if they are small.

Thanks

**Pull request closed without merging**